test/lib/heuristics/scheduling_test.rb

```diff
@@ -553,6 +553,20 @@ def test_sticky_in_scheduling
       refute_includes result[:routes].find{ |r| r[:activities].any?{ |stop| stop[:service_id] == 'service_6_1_1' } }[:vehicle_id], 'vehicle_0_'
     end
 
+    def test_skills_in_scheduling
+      vrp = VRP.lat_lon_scheduling_two_vehicles
+      result = OptimizerWrapper.wrapper_vrp('ortools', { services: { vrp: [:ortools] }}, TestHelper.create(vrp), nil)
+      assigned_route = result[:routes].find{ |r| r[:activities].any?{ |stop| stop[:service_id] == 'service_6_1_1' } }
+      assert_includes assigned_route[:vehicle_id], 'vehicle_0_' # default result
```

Comment:
do we know why vehicle_0_ is "default" when there is no skills? is there a chance that changing the heuristic would assign this service to another vehicle? then this test would give a false negative, right?

Reply:
Maybe we need to add a comment. The aim is exactly that, if we change heuristic such that we do not assign 'vehicle_0_' then we need to change the test. Because skills will not be the reason why we affected this vehicle to this service anymore.